weaver/tests/network-setups/corda/constants.properties

```diff
@@ -1,2 +1,13 @@
-cordaTokenSDKVersion=1.2.5
-cactiVersion=2.0.0-rc.3
\ No newline at end of file
+cordaReleaseGroup=net.corda
+cordaCoreReleaseGroup=net.corda
```

Comment:
Again, I wonder if this file is the right place to specify class or package names rather than `build.gradle`.